### PR TITLE
First build

### PR DESCRIPTION
**Pull request by cbouss**
**Destination channel:** ai-staging

### Explanation of changes:

- To support llama.cpp

### CODE REVIEW

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Comment by danpetry** [comment]
can this go to ai-staging too? If it needs to be turned around today, I don't think there's enough time to do a good enough review to put it on defaults.

**Review by danpetry — COMMENTED** [review]
some changes required to the cbc.yaml
mic_knl (knight's landing) supports AVX512 and should be x86_64_v4
cannon lake likewise
small details tbh, it'll just prevent optimizations on marginal instruction sets

File: abs.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+aggregate_check: false
+
+# destination upon merge
+# not planned for main yet
```
Comment (danpetry) [inline abs.yaml:4]:
```suggestion
# not planned for main yet
# please give a more thorough review if we do want to release to main
```

**Review by danpetry — APPROVED** [review]
I'm fine with this on the basis that it's been done by people who know what they're doing (better than me) and it's going to a staging channel.

**Comment by danpetry** [comment]
I've given it a pretty cursory pass

**Comment by jjhelmus** [comment]
> mic_knl (knight's landing) supports AVX512 and should be x86_64_v4

Knights landing only supports a subset of the AVX-512 instructions needed for x86-64-v4. Specifically it does not implement the VL, DQ, and BW instruction set subsets. 

> cannon lake likewise

This does seem incorrect as cannon lake has the same AVX-512 instruction support as skylake-x which is x86-64-v4. I'll check into it further.

Wikipedia has some nice tables for the x86-64 feature levels, https://en.wikipedia.org/wiki/X86-64#Microarchitecture_levels , and which CPU implement which subsets of AVX-512, https://en.wikipedia.org/wiki/AVX-512#CPUs_with_AVX-512.

**Comment by danpetry** [comment]
thanks for the details and the references.

**Pull request merged**